tests/integration_tests/test_teammatchmaker.py

```diff
@@ -484,3 +484,25 @@ async def test_game_ratings_initialized_based_on_global(lobby_server):
             }
         ]
     }
+
+
+@fast_forward(30)
+async def test_party_cleanup_on_abort(lobby_server):
```

Comment:
Yep exactly. If the party isn't cleaned up, then you'd get a notice message instead of search_info. 

I think I should also specify that `state="start"` in the search_info message.